src/public/index.html

```diff
@@ -17,7 +17,7 @@
     />
     <link
       rel="stylesheet"
-      href="https://use.fontawesome.com/releases/v5.7.0/css/all.css"
+      href="https://use.fontawesome.com/releases/v6.0.0/css/all.css"
```

Comment:
Icon's doesn't appear for some reason

Reply:
@eldemcan this might be a breaking change. Usually they change icon names with major updates. You may want to check from their website how to include their library: https://fontawesome.com/search?new=yes